.circleci/config.yml

```diff
@@ -39,21 +36,17 @@ jobs:
           command: |
             python manage.py check
       - run:
-          name: Black
+          name: Djhtml
           command: |
-            black . --check
-      - run:
-          name: Curly Lint
-          command: | 
-            curlylint .
+            git ls-files '*.html' | xargs djhtml --check
       - run:
           name: Ruff lint
           command: |  
             ruff check .
       - run:
           name: Ruff Format
           command: | 
-            ruff format .
+            ruff format . --check
```

Comment:
Running `ruff format .` in CI isn't really helpful because ruff exits with code 0 even if it changes files, so if files need reformatting, `ruff format .` will just change them and not fail the build.
In CI we need to run `ruff format . --check` which will exit with code 1 if filed need reformatting.